### ISSUE TITLE
List of FTs (Testnet)

### ISSUE THREAD

**Issue by cryptocracy** [cryptocracy/dapparatus#14]
### Assets tab/group

**FTs** (dependent on #5 )
- [x] Wire & Present List of Testnet Fungible Tokens, and their Balance (table view)
- [x] Rig Up FT Transfer Button & Test Transfer on Testnet (transfer from table view to form)
- [x] Create FT Send Form
- [x] Wire Up FT Send Functionality upon Submit Test Transfer (Testnet)

**Comment by cryptocracy** [comment]
Created separate issue to track creation of List of NFTs, this is ok to close now
